integration_tests/src/main/python/conditionals_test.py

```diff
@@ -21,7 +21,15 @@
 import pyspark.sql.functions as f
 
 all_gens = all_gen + [NullGen()]
-all_nested_gens = array_gens_sample + struct_gens_sample
+
+# map_gens_sample have one nested map(map) that cudf will cause bug, will file one bug
+# after bug fixed will use map_gens_sample instead of map_gens_sample_tmp
```

Comment:
```suggestion
# After bug fixed will use map_gens_sample instead of map_gens_sample_tmp
```

Reply:
I think we should wait for the bug being fixed. so it is better to mark this PR as draft.

Reply:
The bug has been fixed in branch-21.10, "map_gens_sample_tmp" is no need now